spec/services/rapid_ready_for_decision/hypertension_pdf_generator_spec.rb

```diff
@@ -29,7 +29,9 @@
     RapidReadyForDecision::HypertensionMedicationRequestData.new(client.list_resource('medication_requests')).transform
   end
 
-  let(:patient_name) { { first: 'Cat', middle: 'Marie', last: 'Power', suffix: 'Jr.' } }
+  let(:patient_name) do
+    { first: 'Cat', middle: 'Marie', last: 'Power', suffix: 'Jr.', va_eauth_birthdate: '10-10-1968' }
```

Comment:
Since you're creating a new hash with whatever keys you want, perhaps use `birthdate` instead of `va_eauth_birthdate`?
```suggestion
    { first: 'Cat', middle: 'Marie', last: 'Power', suffix: 'Jr.', birthdate: '10-10-1968' }
```

Reply:
Okay!